### PR TITLE
User story 21

### LINKED ISSUE #21: User Story 21
Image

As an admin,
When I visit the admin dashboard (/admin)
Then I see the names of the top 5 customers
who have conducted the largest number of successful transactions
And next to each customer name I see the number of successful transactions they have
conducted

### PR DESCRIPTION
**Pull request by westonio**
# Describe the changes below:
- Adds tests and method for Customer model - #top_five_with_most_success_transactions
- Fixes test for Admin Invoice Show page
- Adds tests for Admin Dashboard top 5 customers
- Adds controller action, view, and css for Admin Dashboard top 5 customers
# Relevant user story:
```
As an admin,
When I visit the admin dashboard (/admin)
Then I see the names of the top 5 customers
who have conducted the largest number of successful transactions
And next to each customer name I see the number of successful transactions they have
conducted
```

# Before submitting, check the following:
- [x] Entire test suite is passing
- [99.86%] SimpleCov test percentage

### CODE REVIEW

File: spec/models/customer_spec.rb [hunk 1]
```diff
@@ -11,6 +11,103 @@
...
+        expect(expected_results[2].first_name).to eq(@heber.first_name)
+        expect(expected_results[3].first_name).to eq(@cecelia.first_name)
+        expect(expected_results[4].first_name).to eq(@leanne.first_name)
+      end
+
+      it "has a last name for each customer" do
+        expected_results = Customer.top_five_with_most_success_transactions
+
+        expect(expected_results[0].last_name).to eq(@logan.last_name)
+        expect(expected_results[1].last_name).to eq(@joey.last_name)
```
Comment (dani-wilson) [inline spec/models/customer_spec.rb:96]: really great tests

**Review by dani-wilson — APPROVED** [review]
Good work!!!

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -69,14 +69,27 @@ nav a:hover {
   color: black;
 }
 
-.full-length-header {
+.full-length-header, .half-header {
```
Comment (MWoodshop) [inline app/assets/stylesheets/application.css:72]: Interesting...never heard of a half-header, cool use.

File: app/models/customer.rb
Comment (MWoodshop) [inline app/models/customer.rb:1]: Nice, looks cool. I'll probably need to look at how you formatted it for my next user story. Looks like it adheres to MVC rules too.

File: spec/features/admin/dashboard/index_spec.rb
Comment (MWoodshop) [inline spec/features/admin/dashboard/index_spec.rb:1]: Wow very expansive test!

**Review by MWoodshop — APPROVED** [review]
Looks good!

**Pull request merged**